README.md

```diff
@@ -1,33 +1,15 @@
-            <img src="https://goreportcard.com/badge/github.com/luno/workflow"/> 
-        </a>
-        <a style="padding: 0 5px" href="https://sonarcloud.io/summary/new_code?id=luno_workflow" > 
-            <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=coverage"/> 
-        </a>
-        <a style="padding: 0 5px" href="https://sonarcloud.io/summary/new_code?id=luno_workflow" > 
-            <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=alert_status"/> 
-        </a>
-        <a style="padding: 0 5px" href="https://sonarcloud.io/summary/new_code?id=luno_workflow" > 
-            <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=bugs"/> 
-        </a>
-        <a style="padding: 0 5px" href="https://sonarcloud.io/summary/new_code?id=luno_workflow" > 
-            <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=reliability_rating"/> 
-        </a>
-        <a style="padding: 0 5px" href="https://sonarcloud.io/summary/new_code?id=luno_workflow" > 
-            <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=security_rating"/> 
-        </a>
-        <a style="padding: 0 5px" href="https://sonarcloud.io/summary/new_code?id=luno_workflow" > 
-            <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=sqale_rating"/> 
-        </a>
-        <a style="padding: 0 5px" href="https://sonarcloud.io/summary/new_code?id=luno_workflow" > 
-            <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=vulnerabilities"/> 
-        </a>
-        <a style="padding: 0 5px" href="https://sonarcloud.io/summary/new_code?id=luno_workflow" > 
-            <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=duplicated_lines_density"/> 
-        </a>
+        <img src="https://goreportcard.com/badge/github.com/luno/workflow"/> 
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=coverage"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=alert_status"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=bugs"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=reliability_rating"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=security_rating"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=sqale_rating"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=vulnerabilities"/>
+        <img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=duplicated_lines_density"/>
```

Comment:
I quite liked how it took you to https://sonarcloud.io/summary/new_code?id=luno_workflow when you clicked on the badges! Is it just the display of that you don't like?

Reply:
Okay let me try another way as I liked that too

Reply:
I think i can remove the test decorator...

Reply:
Fixed now with the links remaining 😁